site/src/comparison.rs

```diff
@@ -1248,15 +1238,24 @@ pub enum Direction {
 //
 impl Direction {
     // Also known as the "least upper bound".
-    pub fn join(a: Self, b: Self) -> Self {
-        match (a, b) {
+    pub fn join(self, other: Self) -> Self {
+        match (self, other) {
             (Self::None, b) => b,
             (a, Self::None) => a,
             (Self::Improvement, Self::Improvement) => Self::Improvement,
             (Self::Regression, Self::Regression) => Self::Regression,
             _ => Self::Mixed,
         }
     }
+
+    pub fn overall_result(&self) -> &'static str {
```

Comment:
nit: This function contains text specific to the GH comments, so I think that it should be a function in `comparison_summary`, not a method on `Direction`.